### ISSUE TITLE
Is it possible to hide the date selection when it year selection is open?

### ISSUE THREAD

**Issue by ariesbrunetyson** [720kb/angular-datepicker#40]
When the user open up the year selection from header my date picker looks weird with huge popup. I would like this to be considered for future enhancement. It would give a pleasing look.

**Comment by wouldgo** [comment]
Hi again,

can you firstly post an image reporting this huge popup first (just because I'm curious).
Thanks man.

**Comment by ariesbrunetyson** [comment]
The problem is i have fixed hight page in which the date picker is almost at the end. I was able to bring it in the view when i've date selection alone. But when the year popups up, It is going beyond my page. As you can see the brown in the background is my end on page.

![image](https://cloud.githubusercontent.com/assets/6047770/6371217/6c31ac5c-bd1f-11e4-8035-ed0b4cdaad69.png)

**Comment by wouldgo** [comment]
Hi @ariesbrunetyson 
After a confrontation with @45kb to avoid this issue we have considered to add these two possible functionalities:

- gives the option to hide the "day/month" section when the "year" section is opened;
- put a new attribute that permits the opening position related to the wrapped input (as we already have done in https://github.com/720kb/angular-tooltips)

What do you think about?

Thanks
Dario

**Comment by ariesbrunetyson** [comment]
@wouldgo 
  I completely agree with the first point as it gives option to the developer. But thinking about the second point, I feel like it might not be required as the default behavior for an element with action popup is to appear below the element.

**Comment by chadfawcett** [comment]
Hello @wouldgo . I would also think it would be nice to have the year selection auto hide after selecting the year. Similar to how the whole input hides once a date is picked.

I think a nice workflow would be to have it open with the month calendar showing like it normally does. If I need to change the year, I click on the year at the top, the year selection opens up, the date selection hides (as @ariesbrunetyson mentioned) and then once I select the new year, the year selection hides and the date selection shows again.

How do you feel about that?

I would be willing to do the code for this auto hiding year feature as well as the auto hiding date feature. I think the position feature could be its own PR if you in fact decide to implement it. I would just expect it to drop dow below the input as that seems pretty common.

**Comment by sravank** [comment]
Hello Sir, 

 I am facing small issue how to disable feature dates please advice us.

**Comment by ariesbrunetyson** [comment]
@sravank Did you check `date-max-limit=""` attribute?

**Comment by NewSonu** [comment]
I have fixed this issue, angular-datepicker.js file updated code below :

Live Example:
https://plnkr.co/edit/a2s7e3uwIgFhtyqEnaVV?p=info
